pkg/kubectl/cmd/logs.go

```diff
@@ -71,7 +71,7 @@ var (
 )
 
 const (
-	logsUsageStr = "expected 'logs (POD | TYPE/NAME) [CONTAINER_NAME]'.\nPOD or TYPE/NAME is a required argument for the logs command"
+	logsUsageStr = "expected 'logs [-f] [-p] (POD | TYPE/NAME) [-c CONTAINER]'.\nPOD or TYPE/NAME is a required argument for the logs command"
```

Comment:
Perhaps, you can reuse the string here.
https://github.com/kubernetes/kubernetes/blob/c4ca6638c397e90a486d363a10b3e6411fa6a0e4/pkg/kubectl/cmd/logs.go#L103

You do not have to modify multiple strings if the usage is changed again.